Block/BaseBlock.php

```diff
@@ -127,6 +127,9 @@ public function getConfigs()
         $currencyCode = $this->storeManager->getCurrentStoreCurrency()->getCode();
         $currencySymbol = $this->priceCurrency->getCurrencySymbol();
 
+        $filtersAndSortIconPosition = $this->storeSearchConfig->getFiltersAndSortIconPosition()
```

Comment:
Keep this directly in getFiltersAndSortIconPosition function. It will either return the value if set or default value.